deploy/pgbouncer.ini

```diff
@@ -0,0 +1,18 @@
+[databases]
+* = host=postgres port=5432
```

Comment:
I'm still a bit lost at how services meshes together.
Is is:
```
Internet -> Omaha -> PGBouncer -> Postgres
```
or 
``` 
Internet -> Omaha -> Postgres <- PGBouncer
````

Reply:
the first one